src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./index.less";
+import LogoPng from "images/MFLOGOTEXT.png";
+
+const Navbar = () => {
+  return (
+    <header>
+      <nav class="nav">
+        <div>
+          <Link class="nav__brand" to="/">
```

Comment:
change the class to className in whole project

Reply:
thanks i missed it